agent/app/args/flag.go

```diff
@@ -13,11 +13,14 @@
 
 package args
 
-import "flag"
+import (
+	"flag"
+)
 
 const (
 	versionUsage             = "Print the agent version information and exit"
-	logLevelUsage            = "Loglevel: [<crit>|<error>|<warn>|<info>|<debug>]"
+	driverLogLevelUsage      = "Loglevel for docker logging driver: [<crit>|<error>|<warn>|<info>|<debug>]"
+	instanceLogLevelUsage    = "Loglevel for Agent on-instance file: [<none>|<crit>|<error>|<warn>|<info>|<debug>]"
```

Comment:
nit: can we have "Agent on-instance log file" instead of "Agent on-instance file"?